app/src/main/res/values/strings.xml

```diff
@@ -128,6 +128,13 @@
     <string name="loading">Loading…</string>
     <string name="you_should_choose_image">You should choose image!</string>
     <string name="blocked">Blocked</string>
-    <string name="active">Active</string>
     <string name="wait">Wait</string>
+    <string name="active">active</string>
+    <string name="inactive">inactive</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.inactive` appears to be unused
 Unused resources make applications larger and slow down builds.